roles/contiv_network/tasks/main.yml

```diff
@@ -1,10 +1,25 @@
 ---
 # This role contains tasks for configuring and starting netmaster and netplugin service
 
+- name: remove older netmaster
```

Comment:
Unfortunately, contivnet binaries do not support version checks now. Therefore, I cannot implement this using facts they way I did for docker and swarm. But, this can definitely be done in the future.